gsa/align.go

```diff
@@ -15,6 +15,31 @@ package gsa
 //      The two rows in the pairwise alignment
 func Align(p, q, edits string) (pRow, qRow string) {
 	pRow, qRow = "", ""
+	pItr, qItr := 0, 0
+	for pos, char := range edits {
+		char := string(char)
```

Comment:
If you use `'M'`, `'I'` and `'D'` instead of `"M"`, `"I"`, and `"D"` you don't have to cast the type. Right now, you are using double quotes, so you have strings, and then `char` must be a string. But if it is a `byte`, you can compare it against the operations in single quotes.